src/action/payment.js

```diff
@@ -199,6 +220,27 @@ class PaymentAction {
     }
   }
 
+  /**
+   * Estimate the lightning transaction fee using the queryRoutes grpc api
+   * after which the fee is set in the store.
+   * @param  {number} options.satAmt      The amount to be payed in satoshis
+   * @return {Promise<undefined>}
+   */
+  async reEstimateLightningFee() {
+    try {
```

Comment:
I wonder if the error from this try/catch block should be propagated further from this method so that the UI can display some useful info to the use and try to recover rather than just having a log message.

Reply:
Thinking about it, that exception will only be thrown if we do not have a valid lightning invoice in the address field. That is dependent on other functions being called before this one. That kind of subtle and implicit dependencies are dangerous, IMHO. I have tried to make the expectations of the method more clear both in code and documentation. 

PS: I am used to writing functional Typescript code in my day job, so all this state and no typing is making me a bit uncomfortable.

Reply:
Fixed in 93227257 and 51413f29